apps/src/blockly/addons/cdoTheme.js

```diff
@@ -5,6 +5,9 @@ export default GoogleBlockly.Theme.defineTheme('modern', {
   blockStyles: {
     colour_blocks: {
       colourPrimary: '#0093c9'
+    },
+    procedure_blocks: {
+      colourPrimary: '#509918'
```

Comment:
When this file was started, we didn't import constants for these because they weren't used anywhere else in the repository: https://github.com/code-dot-org/code-dot-org/pull/43197

Reply:
Gotcha - thanks for the context.